### ISSUE TITLE
`markdown` no longer exports version

### ISSUE THREAD

**Issue by jonringer** [radude/mdx_truly_sane_lists#11]
Disclaimer:
I'm aware that the version used in this package may be outside of the expect range of package versions. However, python requires a single version of a given package in any runtime environment. This issue is meant to be more of a "inform upstream that there is a [potential] issue" rather than "we expect upstream to immediately react to the issue".

Issue:
In mardown>3.4, it no longer exports a version
```
Python 3.10.5 (main, Jun  6 2022, 12:05:50) [GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import markdown
>>> markdown.
markdown.Extension(         markdown.core               markdown.markdown(          markdown.serializers
markdown.Markdown(          markdown.extensions         markdown.markdownFromFile(  markdown.treeprocessors
markdown.blockparser        markdown.htmlparser         markdown.postprocessors     markdown.util
markdown.blockprocessors    markdown.inlinepatterns     markdown.preprocessors
```

Which invalidates the package:
```
  File "/build/source/mdx_truly_sane_lists/mdx_truly_sane_lists.py", line 8, in <module>
    from markdown import version as md_version
ImportError: cannot import name 'version' from 'markdown' (/nix/store/51wn12kvpm6jmkfjnl88z10780is676d-python3.10-markdown-3.4.1/lib/python3.10/site-packages/markdown/__init__.py)
```

**Comment by jonringer** [comment]
https://python-markdown.github.io/change_log/release-3.4/ looks like the proper thing is now `markdown.__version__`

**Comment by xmatthias** [comment]
i guess that's a duplicate of #9 ... which comes to the same conclusion - and has a (potential) fix available in #10.
